pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel.go

```diff
@@ -118,6 +118,7 @@ func (r *Reconciler) reconcile(ctx context.Context, imc *v1alpha1.InMemoryChanne
 	}
 
 	imc.Status.SubscribableTypeStatus.SubscribableStatus = r.createSubscribableStatus(imc.Spec.Subscribable)
+	imc.Status.SubscribableTypeStatus.SubscribableStatusV2 = r.createSubscribableStatus(imc.Spec.Subscribable)
```

Comment:
I would feel more comfortable if we set both of these at the same time with a method.

```golang
imc.Status.SetSubscribableTypeStatus(r.createSubscribableStatus(imc.Spec.Subscribable))
```

Then have a function on Subscribable type:

```golang
func (s SubscribableTypeStatus) SetSubscribableTypeStatus(subscriberStatus SubscriberStatus) {
    s.SubscriberStatus = &subscriberStatus
    s.DeprecatedSubscriberStatus = &subscriberStatus
}
```

Reply:
done